mk/docker.mk

```diff
@@ -32,7 +32,9 @@ docker-login:
 
 .PHONY: docker-build
 docker-build:  ## Build image DOCKER_IMAGE from DOCKER_DOCKERFILE using the DOCKER_CONTEXT_DIR
-	$(DOCKER) build $(DOCKER_BUILD_OPTS) -t "$(DOCKER_IMAGE)" -f $(DOCKER_DOCKERFILE) $(DOCKER_CONTEXT_DIR)
+	# spooky action at a distance
+	ln -f $(DOCKER_DOCKERFILE) Containerfile
+	$(DOCKER) build $(DOCKER_BUILD_OPTS) -t "$(DOCKER_IMAGE)" $(DOCKER_CONTEXT_DIR)
```

Comment:
can you add Containerfile to .gitignore?

Reply:
or just rm it after the build?

Reply:
gitignore'd it, thanks!